ui/core/_individual_sim_ui.scss

```diff
@@ -130,6 +130,24 @@
 .settings-section-container {
   position: absolute;
 }
+
+.player-row {
```

Comment:
Don't add css rules in core unless you're sure; this affects everyone! You can override specific things by putting these same rules in `ui/warlock/_sim.ts`

Reply:
I played a bit with the UI, and then changed my mind, and forgot to remove this part, whoops sorry!